Jamfile

```diff
@@ -491,6 +491,7 @@ variant test_release : release
 	: <asserts>production <debug-symbols>on
 	  <invariant-checks>full <boost-link>shared <optimization>off
 	  <export-extra>on <debug-iterators>on <threading>multi
+	  <inlining>off
```

Comment:
oh. you're right about the optimizations.. So, by upgrading the image, you get a newer clang version, but you don't get a newer boost-build version, so that causes problems, is that the case?